src/ui/components/CarouselSection/icons/default.svg

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
extra blank line should be removed.

Reply:
True, this was just copy/pasted from Sketch.